lib/linguist/generated.rb

```diff
@@ -248,6 +249,16 @@ def generated_protocol_buffer?
 
       return lines[0].include?("Generated by the protocol buffer compiler.  DO NOT EDIT!")
     end
+    
+    # Internal: Is the blob generated by Apache Thrift compiler?
+    #
+    # Returns true or false
+    def generated_apache_thrift?
+      return false unless ['.rb', '.py', '.go', '.js', '.m', '.java', '.h', '.cc', '.cpp'].include?(extname)
+      return false unless lines.count > 1
+
+      return lines[0].include?("Autogenerated by Thrift Compiler") || lines[1].include?("Autogenerated by Thrift Compiler")
```

Comment:
@arfon @bkeepers Should we change this to `lines[0..1].join("").include?("Autogenerated by Thrift Compiler")`?